it/test/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/controllers/AccountingPeriodControllerISpec.scala

```diff
@@ -24,6 +24,7 @@ import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.helpers.{ComponentSpecBase,
 import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.models.AccountingPeriod.{FirstAprilToThirtyFirstMarch, OtherAccountingPeriod, SixthAprilToFifthApril}
 import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.pages.AccountingPeriodPage
 import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.views.PageContentBase
+import org.mongodb.scala.SingleObservableFuture
```

Comment:
We could move the collection.drop methods which use `toFuture` to the `DatabaseHelper `trait? Then we won't need this import in all ISpecs and just have 1 reusable method. Just an idea, maybe for later